config/client.json

```diff
@@ -1,4 +1,5 @@
 [
+	"anchor-fm",
```

Comment:
I don't think this is needed, since the new `anchor-fm` config is a boolean value acting as a feature flag that can be checked with `config.isEnabled( <PROPERTY_NAME> )`.

This file should contain only configs whose values need to be retrieved with `config( <PROPERTY_NAME> ) `: https://github.com/Automattic/wp-calypso/blob/master/config/README.md

> If it is necessary to access a config value on the client-side, add the property name to the client.json file, which is list of specific config properties that will be exposed to the client.